command.go

```diff
@@ -370,6 +371,9 @@ func (c *Command) Find(arrs []string) (*Command, []string, error) {
 				// only accept a single prefix match - multiple matches would be ambiguous
 				if len(matches) == 1 {
 					return innerfind(matches[0], argsMinusX(args, argsWOflags[0]))
+				} else if len(matches) == 0 && len(args) > 0 && args[0] == "help" {
+					// special case help command
+					return innerfind(c, argsMinusX(append(args, "--help"), argsWOflags[0]))
```

Comment:
So this is interesting.  Before this change.

command help subcommand - works, shows help for subcommand
command subcommand help - fails, because 'help' is an unknown subsubcommand 

Are we certain we want the later?  (I personally like it)